vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -5,10 +5,13 @@ export API_ENDPOINT_PORT="6443"
 export IMAGE_URL='{{ IMAGE_URL }}'
 export IMAGE_CHECKSUM='{{ IMAGE_CHECKSUM }}'
 export CTLPLANE_KUBEADM_EXTRA_CONFIG="
+    users:
+    - name: metal3
+      sudo: "ALL=(ALL) NOPASSWD:ALL"
```

Comment:
The CI is failing because of the quotes. You need to change the type of quotes or escape them

Reply:
Thanks updated